thunderbird/default.cfg

```diff
@@ -17,4 +17,4 @@ parallelprocesses = 2
 [logging]
 level = INFO
 file = thunderbird.log
-format = %(asctime)s] [%(levelname)s] line=%(lineno)s module=%(module)s %(message)s
+format = [%(asctime)s] [%(levelname)s] line=%(lineno)s module=%(module)s %(message)s
```

Comment:
Has this changed anything? Do you think the missing `[` was a mistake from the original `cookiecutter`? If so, I can let Long know so that it can be fixed.

Reply:
@nikola-rados  I just checked `cookiecutter`, and I believe it has the same [mistake](https://github.com/bird-house/cookiecutter-birdhouse/blob/master/%7B%7Bcookiecutter.project_repo_name%7D%7D/%7B%7Bcookiecutter.project_slug%7D%7D/default.cfg#L20). I know it is a mistake because missing the opening bracket logs out a string without the missing bracket, which I think is not what we want.